meilisearch/_httprequests.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, config: Config) -> None:
         self.config = config
         self.headers = {
             "Authorization": f"Bearer {self.config.api_key}",
-            "User-Agent": qualified_version(),
+            "User-Agent": qualified_version()
+            + (";" + ";".join(config.client_agents) if config.client_agents else ""),
```

Comment:
I'm not very satisfied with this line, if you know a better way, @sanders41  maybe, please lead the way!

Reply:
Does this work?

```
"User-Agent": ";".join(config.client_agents.append(qualified_version()))
```

Just ensure the `client_agents` is at least a `[]` array

Reply:
Right now the user agent is a string like: `Meilisearch Python (v0.26.0);Meilisearch Package1 (v1.1.1);Meilisearch Package2 (v2.2.2)`

Are you saying you want `[Meilisearch Python (v0.26.0), Meilisearch Package1 (v1.1.1), Meilisearch Package2 (v2.2.2)]`?

Reply:
I think he meant the client's input parameter, making adding several headers possible.

Reply:
The outcome should be a string, but until the last part (sending the data to the header) we could work with an array.

Reply:
Ok, if I'm understanding correctly this should be good then. The user specifies it as a tuple like `("Meilisearch Package1 (v1.1.1)", "Meilisearch Package2 (v2.2.2)")` and can add as many as needed.